spec.bs

```diff
@@ -5367,6 +5382,8 @@ To <dfn>check interest group permissions</dfn> given an [=origin=] |ownerOrigin|
   ::  "`omit`"
   :   [=request/redirect mode=]
   :: "`error`"
+  :   [=request/policy container=]
+  ::  "`client`"

```

Comment:
Do we still want this for permissions check?

Reply:
We're passing in a null client here, still, which would mean no checks (which I think is a bug - we want the real client here, or we need to pass in the container, or at least the IP address space stuff)

Reply:
we can remove it, since unset is the same as being "`client`" https://fetch.spec.whatwg.org/#concept-request-policy-container. And when "`client`" is null, policy container will be set to a new policy container (step 12 of https://fetch.spec.whatwg.org/#concept-fetch).

Reply:
> We're passing in a null client here, still, which would mean no checks (which I think is a bug - we want the real client here, or we need to pass in the container, or at least the IP address space stuff)

Are you saying that the expected behaviour would be similar to other requests that has the frame's policy container's IP address space?

Reply:
Well, for this one, I think we may want to treat it as if it were made by the actual frame as a normal JS request - that's what we currently do (though we bypass service worker).  This is cross-origin request, with no previous approvals to make a cross origin request, unlike the bidder script.

We can certainly talk about changing behavior, but currently we use the full container properties and everything else the frame provides that's implemented in net/ for this request, rather than just copying the ClientSecurityState or the frame's policy container.

Reply:
Changed its client to the settings environment object. Changed its origin and policy container to "client", which can be removed since that's the default value when not set, but I'm still keeping it (no preference of whether keeping it or not). Setting them to "client" will correctly use "client"'s origin and policy container, see their [notes](https://fetch.spec.whatwg.org/#concept-request-origin), and step 10 and 12 of [fetching algorithm](https://fetch.spec.whatwg.org/#concept-fetch).